specs/category.spec.ts

```diff
@@ -0,0 +1,63 @@
+describe('InstantSearch - Search on specific category', () => {
```

Comment:
general question, but isn't it always going to be prefixed with "InstantSearch"?

Reply:
Since the SauceLabs account is used by other teams too then I prefixed all our test names with `InstantSearch`